jf: add page for JFrog CLI (#10471)
tldr-pages/tldr: pages/common/jf.md

# jf

> > Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.

Reviewer: ```suggestion
> Interact with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```

> > Use JF CLI for interfacing with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.

Reviewer 2: ```suggestion
> Interface with JFrog products like Artifactory, Xray, Distribution, Pipelines and Mission Control.
```
How about this? (As we are currently moving from the usage of `CLI` term in page descriptions)

Author: oh that's a good point. I like your suggestion

> More information: <https://jfrog.com/help/r/jfrog-cli/usage>.

> - Add new configuration:

Reviewer: ```suggestion
- Add a new configuration:
```

`jf config add`

> - Show current configuration:

Reviewer: ```suggestion
- Show the current configuration:
```

`jf config show`

> - Search artifacts within the given repository and directory:

Reviewer: ```suggestion
- Search for artifacts within the given repository and directory:
```

> - Artifactory: Search artifacts within given repository and directory:

Reviewer 2: ```suggestion
- Search artifacts within the given repository and directory:
```

Author: thank you for the suggestions
